fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend

```diff
@@ -204,6 +204,8 @@ class «cls.name»(«cls.inheritedPythonClassNameHL»):
 «cls.description.description»
 """
 
+«cls.protectedAreaHL("system_imports")»

```

Comment:
We need to add a check for "protected regions" enabled here too.

Reply:
Thx, good hint. Done!

Reply:
Can we factorize the check in the protectedAreaHL method ?

Reply:
@dlacoste-esrf Do you mean can we move the check for "protected regions enabled" inside `protectedAreaHL`?  That sounds like good idea, if it possible.

Reply:
exactly.

Reply:
@dlacoste-esrf  thanks for your improvements, looks nice.